File: backend/Models/Brand.cs
namespace influenco.backend.Models;

public class Brand : BaseEntity
{
    public Guid AppUserId { get; set; }
    public AppUser AppUser { get; set; } = null!;

    public string CompanyName { get; set; } = null!;
    public string? Description { get; set; }
    public string? LogoUrl { get; set; }
    public string? Website { get; set; }
    public string? Industry { get; set; }
}

File: frontend/src/api/types/brand.ts
export interface BrandProfile {
    id: string;
    companyName: string;
    description?: string;
    logoUrl?: string;
    website?: string;
    industry?: string;
    email: string;
    activeCampaignsCount: number;
    dealsCount: number;
}

export interface UpdateBrandProfileRequest {
    companyName: string;
    description?: string;
    logoUrl?: string;
    website?: string;
    industry?: string;
}